terraform/variables.tf

```diff
@@ -77,15 +93,37 @@ locals {
   app_secrets       = yamldecode(data.azurerm_key_vault_secret.app_secrets.value)
   infra_secrets     = yamldecode(data.azurerm_key_vault_secret.infra_secrets.value)
 
-  app_env_values  = try(yamldecode(file("${path.module}/workspace-variables/${var.paas_app_environment}_app_env.yml")), {})
+  app_env_values_from_yaml = try(yamldecode(file("${path.module}/workspace-variables/${var.paas_app_environment}_app_env.yml")), {})
+
+  app_env_values = merge(
+    local.app_env_values_from_yaml,
+    { DB_SSLMODE = var.db_sslmode }
+  )
 
-  custom_domain = { "CUSTOM_HOSTNAME" = "apply-${local.app_name_suffix}.london.cloudapps.digital" }
+  custom_domain    = { "CUSTOM_HOSTNAME" = "apply-${local.app_name_suffix}.london.cloudapps.digital" }
   authorized_hosts = { "AUTHORISED_HOSTS" = "apply-${local.app_name_suffix}.london.cloudapps.digital" }
 
   paas_app_environment_variables = merge(
     local.custom_domain,
     local.authorized_hosts,
-    local.app_secrets, # Values in app secrets can override anything before it
+    local.app_secrets,   # Values in app secrets can override anything before it
     local.app_env_values # Utilimately app_env_values can override anything in the merged map
   )
+  cluster = {
```

Comment:
Future improvement: this works for now but copying and maintaining this across multiple repos probably won't scale well.  Not sure how to simplify it but we should definitely look at refactoring

Reply:
Good point. Ideally this data will be in a shared module.

Reply:
https://trello.com/c/aExRfBEy